### PR TITLE
Backend: MarkMarker: update how we manage training data

### PR DESCRIPTION
**Pull request by helicopter88**
The table used in this PR is temporary, should be switched to the one we're using right now soon

### CODE REVIEW

File: backend/mark_marker/src/main/scala/mark_marker/trainer/DatabaseConnector.scala [hunk 1]
```diff
@@ -1,96 +1,55 @@
...
 import scala.io.Source
 
-class DatabaseConnector {
+class DatabaseConnector(dbName: String) {
   private val driver = "com.mysql.cj.jdbc.Driver"
-  private val url = "jdbc:mysql://tc.jstudios.ovh:3306/Automarker"
   private val username = "python"
-  private val psw = findPsw("dbpass.txt")
+  private val url = "jdbc:mysql://tc.jstudios.ovh:3306/Automarker"
+  private val psw = findPsw("../dbpass.txt")
```
Comment (helicopter88) [inline backend/mark_marker/src/main/scala/mark_marker/trainer/DatabaseConnector.scala]: should be dbpass.txt

Comment (giuliojiang) [reply backend/mark_marker/src/main/scala/mark_marker/trainer/DatabaseConnector.scala]: when a microservice is launched, the working dir is alabno
so it should be `dbpass.txt`

**Review by giuliojiang — COMMENTED** [review]
database with line by line exercises is good

does mark marker download the training set every time?
where does it store the training data / serialized training data?

**Comment by helicopter88** [comment]
Done, please review

**Review by giuliojiang — APPROVED** [review]

**Review by procaconsul — APPROVED** [review]

File: infrastructure/infrastructure/src/main/java/alabno/msfeedback/Runner.java [hunk 2]
```diff
@@ -1,25 +1,19 @@
...
-    HaskellMarkerRunner(HaskellMarkerUpdater updater) {
+    public Runner(MicroServiceUpdater updater) {
         this.updater = updater;
     }
-    
+
     @Override
     public void run() {
-        System.out.println("HaskellMarkerRunner: starting...");
+        System.out.println("Runner starting for: " + updater.getClass());
```
Comment (eliasbenussi) [inline infrastructure/infrastructure/src/main/java/alabno/msfeedback/Runner.java:16]: cheeky

File: infrastructure/infrastructure/src/main/java/alabno/msfeedback/markmarker/MarkMarkerUpdater.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        Thread updaterThread = new Thread(new Runner(this));
+        updaterThread.start();
+    }
+
+    @Override
+    public void update(SourceDocument source, int lineNumber, String type, String annotation) {
+        // Does nothing
+    }
+
+    @SuppressWarnings("unchecked")
```
Comment (eliasbenussi) [inline infrastructure/infrastructure/src/main/java/alabno/msfeedback/markmarker/MarkMarkerUpdater.java:47]: what does this do?

Comment (helicopter88) [reply infrastructure/infrastructure/src/main/java/alabno/msfeedback/markmarker/MarkMarkerUpdater.java:47]: Suppresses a warning triggered by a downcast that I have down

**Pull request merged**